code/core/src/core-server/utils/new-story-templates/typescript.ts

```diff
@@ -22,7 +22,7 @@ export async function getTypeScriptTemplateForNewStoryFile(data: TypeScriptTempl
     : `import { ${importName} } from './${data.basenameWithoutExtension}'`;
 
   return dedent`
-  import type { Meta, StoryObj } from '${data.rendererPackage}';
+  import type { Meta, StoryObj } from '${data.frameworkPackage}';
```

Comment:
Will this work with community frameworks? 🤔

Reply:
Which react-based community frameworks do you have in mind? Do any exist?

Reply:
I think in Storybook 9, it is a requirement that community frameworks, which use one of our renderers as basis, have to re-export them. That's what we are doing, and that's what addon/framework authors should do as well.

Reply:
I'm thinking of rspack, but I guess it will work as well